cmd/drand-cli/cli.go

```diff
@@ -349,7 +349,7 @@ var storageTypeFlag = &cli.StringFlag{
 var pgDSNFlag = &cli.StringFlag{
 	Name:    "pg-dsn",
 	Usage:   "PostgreSQL DSN configuration.",
-	Value:   "postgres://drand:drand@localhost:5432/drand?sslmode=disable&timeout=5&connect_timeout=5",
+	Value:   "postgres://drand:drand@localhost:5432/drand?sslmode=disable&connect_timeout=5",
```

Comment:
It might be good to comment / explain each "supported" options. Are there more that these 2 ones?

Reply:
Currently, these are the only two supported options. I've added more documentation for them, good idea!